core/src/OpenACC/Kokkos_OpenACC_ParallelScan_Range.hpp

```diff
@@ -88,22 +86,27 @@ class ParallelScanOpenACCBase {
         functor(m_functor);
     const IndexType N        = end - begin;
     const IndexType n_chunks = (N + chunk_size - 1) / chunk_size;
+#ifdef KOKKOS_COMPILER_CLANG
+    int const num_elements = n_chunks * 2 * chunk_size;
+#else
+    int const num_elements = 2 * chunk_size;
```

Comment:
I was excepting you would use `num_elements` in the definition of `KOKKOS_IMPL_ACC_ELEMENT_VALUES_CLAUSE`
Not blocking though

Reply:
Updated.